### DISCUSSION TITLE
How to decrease Supabase JWT session size?

### DISCUSSION THREAD

**Discussion by remorses** [supabase/supabase#28259] [Questions]
I want to store the Supabase session for a plugin and i am constrained to a total of 4kb, is there a way to decrease the Supabase session JWT? Maybe deleting the `raw_user_meta_data `? What is the recommended way?

**Comment by GaryAustin1** [comment]
The only impact you can have is raw user meta data and maybe app meta data.  The problem is some info may get added again if the user signs up with a new provider, but you could always have a trigger to null it out if updated.
I don't know if supabase uses the app metadata internally or not though versus just the identities table.
